File: app/src/main/java/com/ninhttd/devtest/presentation/product/model/Rating.java

package com.ninhttd.devtest.presentation.product.model;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class Rating {

    @SerializedName("averagePoint")
    private Object mAveragePoint;
    @SerializedName("voteCount")
    private Object mVoteCount;

    public Object getAveragePoint() {
        return mAveragePoint;
    }

    public void setAveragePoint(Object averagePoint) {
        mAveragePoint = averagePoint;
    }

    public Object getVoteCount() {
        return mVoteCount;
    }

    public void setVoteCount(Object voteCount) {
        mVoteCount = voteCount;
    }

}

File: app/src/main/java/com/ninhttd/devtest/data/entity/Rating.kt
package com.ninhttd.devtest.data.entity

import com.google.gson.annotations.SerializedName

class Rating {

    @SerializedName("averagePoint")
    var averagePoint: Any? = null
    @SerializedName("voteCount")
    var voteCount: Any? = null

}
